test cases/common/16 configure file/meson.build

```diff
@@ -86,3 +86,15 @@ foreach config_template : config_templates
 endforeach
 
 test('Substituted', executable('prog4', 'prog4.c'))
+
+# Test `capture` keyword
+
+configs = [
+    # no input
+    configure_file(command: [ find_program('echo'), 'hello', 'world'], capture: true, output: 'capture.test1'),
+    # with input
+    configure_file(input: 'hello world', command: [ find_program('basename'), '@INPUT@'], capture: true, output: 'capture.test2'),
+]
+foreach c : configs
+    test('@0@'.format(c), find_program('grep'), args: ['-q', '-e', '^hello world$', c])
```

Comment:
This should not use grep or echo because the tests should run without any dependency on a specific userland. Grep is not available on stock Windows and echo works differently in different shells (see 38 run program for example). Our CI environment apparently only has machines that have mingw/equiv installed so these tests pass.

These programs should be standalone Python scripts having equivalent functionality.